src/DefaultMessageOptions.cmake

```diff
@@ -7,4 +7,5 @@ set_default(FUNCTION_ARGUMENT_NAME_SIZE 16)
 set_default(BUZZ_PAYLOAD_SIZE 16)
 
 set_default(BYTES_PAYLOAD_SIZE 200)
-set_default(NEIGHBORS_LIST_SIZE 50)
\ No newline at end of file
+set_default(NEIGHBORS_LIST_SIZE 50)
+set_default(AGENTS_LIST_SIZE 50)
```

Comment:
Shouldn't we make Neighbors list size = Agents list - 1?

Reply:
Fixed, and set to agent_list since it's the max, so it will take the same space in memory